### PR TITLE
[4] composer update willdurand/negotiation from 2.3 to 3.0

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #32571 .

### Summary of Changes
willdurand/negotiation from 2.3 to 3.0 needed for php 8

### Testing Instructions
see #32571

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -753,16 +753,16 @@
         },
         {
             "name": "joomla/console",
-            "version": "2.0.0-beta3",
+            "version": "2.0.0-beta2",
```
Comment (richard67) [inline composer.lock]: Downgrade joomla/console from 2.0.0-beta3 to 2.0.0-beta2? Why that?

File: composer.lock [hunk 2]
```diff
@@ -1143,16 +1133,16 @@
         },
         {
             "name": "joomla/filter",
-            "version": "2.0.0-beta5",
+            "version": "2.0.0-beta2",
```
Comment (richard67) [inline composer.lock]: Downgrade joomla/filter from 2.0.0-beta5 to 2.0.0-beta2? Why that?

File: composer.lock [hunk 3]
```diff
@@ -1163,7 +1153,7 @@
                 "joomla/language": "<2.0"
             },
             "require-dev": {
-                "joomla/coding-standards": "^3.0@dev",
+                "joomla/coding-standards": "^2.0@alpha",
```
Comment (richard67) [inline composer.lock]: This unwanted changed seems to be caused by one of the previous unwanted changes (downgrades).

File: composer.lock (hunk 1 above)
Comment (alikon) [reply composer.lock]: i've started with an old branch 😜

Comment (alikon) [reply composer.lock]: fixed

File: composer.lock (hunk 3 above)
Comment (alikon) [reply composer.lock]: fixed

File: composer.lock (hunk 2 above)
Comment (alikon) [reply composer.lock]: fixed

**Review by richard67 — APPROVED** [review]

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**